backend/.gitignore

```diff
@@ -7,6 +7,7 @@ pom.xml.asc
 *.class
 /.lein-*
 /.nrepl-port
+/.dir-locals.el
```

Comment:
there are some double entries in this `.gitignore` and the toplevel one. Perhaps we should have only one toplevel ignore file?